packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/AltAccountInfo.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { userGetProfileUrl } from '../../../lib/collections/users/helpers';
+import { useMulti } from '../../../lib/crud/withMulti';
+import { useSingle } from '../../../lib/crud/withSingle';
+import { Link } from '../../../lib/reactRouterWrapper';
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import LockIcon from '@material-ui/icons/Lock'
+import LockOpenIcon from '@material-ui/icons/LockOpen'
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+
+  },
+  icon: {
+    width: 12,
+    height: 12,
+    cursor: "pointer",
+    color: theme.palette.grey[500],
+    marginLeft: 2,
+    marginTop: 1
+  }
+});
+
+export const AltAccountInfo = ({classes, user}: {
+  classes: ClassesType,
+  user: SunshineUsersList
+}) => {
+  const [showAlternateAccounts, setShowAlternateAccounts] = useState<boolean>(false)
+  const { Loading, LWTooltip } = Components
+
+  const { results, loading } = useMulti({
+    terms: {
+      view: "usersByUserIds",
+      userIds:  user.associatedClientIds?.[0].userIds
```

Comment:
This is wrong; you need to use all the associated client IDs, not just the first one.